runtime/rasdump/javadump.cpp

```diff
@@ -5796,3 +5863,122 @@ getAllocatedVMThreadCount(J9JavaVM *vm)
 
 	return count;
 }
+
+/**
+ * This function is used to get the Redhat OS distribution details
+ * @param [out] OS Distribution Name
+ *
+ * @return returns 0 on success, any other -1 code on failure
+ */
+static int 
+getRedhatDistributionInfo(char *strDistributionVersion)
+{
+	int nSuccess = -1;
+	char strLine [OPENJ9_VERSION_INFO_SIZE] = "";
+	FILE *fpVersion = NULL;
+	/*read redhat Os distribution info file */
+	fpVersion = fopen(OPENJ9_REDHAT_OS_DISTRIBUTION_INFO_FILE, "r");
```

Comment:
Please use OMR support API to make the code more portable:
```
fopen  -> omrfile_open
fgets  -> omrfile_read
fclose -> omrfile_close
```
The file must be closed regardless of whether the read was successful.